src/rad/resources/schemas/aperture-1.0.0.yaml

```
@@ -3,11 +3,14 @@
 $schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
 id: asdf://stsci.edu/datamodels/roman/schemas/aperture-1.0.0
 
-title: Aperture information
+title: Aperture Information
 type: object
 properties:
   name:
-    title: PRD science aperture used
+    title: PRD Science Aperture Used
+    description: |
```

```suggestion
    description: >-
```
What are your thoughts on using a [folded block with block chomping](https://yaml-multiline.info/) instead of a literal block so that the newlines in the description are stripped on read?

Using literal blocks `|` the `schema_info` contains newlines:
```python
> af.schema_info()['roman']['meta']['aperture']['name']['description'].info
'Science aperture used as defined in the Project\nReference Database (PRD).\n'
```
Switching to a folded block with block chomping `>-` strips the newlines:
```python
> af.schema_info()['roman']['meta']['aperture']['name']['description'].info
'Science aperture used as defined in the Project Reference Database (PRD).'
```

---

Honestly it does not matter to me. That sort of detail is up to the RAD and Roman pipeline developers.

---

Yup, switching `|` to `>` would replace the internal new line characters with spaces (`>-` would also strip the trailing newline). This also applies to `title` and possibly other descriptive fields.

---

I think this is a discussion for a separate set of changes. It can happen before these changes are merged, in which case this PR can be updated to match that style, or after this PR. However, the specifics of which to use distracts from the main point of these changes, which are to update the schemas with the titles and descriptions provided by INS.